src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -37,25 +42,23 @@ public static void SetCurrentCircuitHost(CircuitHost circuitHost)
         {
             _current.Value = circuitHost ?? throw new ArgumentNullException(nameof(circuitHost));
 
-            Microsoft.JSInterop.JSRuntime.SetCurrentJSRuntime(circuitHost.JSRuntime);
+            JSInterop.JSRuntime.SetCurrentJSRuntime(circuitHost.JSRuntime);
             RendererRegistry.SetCurrentRendererRegistry(circuitHost.RendererRegistry);
         }
 
         public event UnhandledExceptionEventHandler UnhandledException;
 
-        private bool _isInitialized;
-        private Action<IComponentsApplicationBuilder> _configure;
-
         public CircuitHost(
             IServiceScope scope,
             IClientProxy client,
             RendererRegistry rendererRegistry,
             RemoteRenderer renderer,
             Action<IComponentsApplicationBuilder> configure,
             IJSRuntime jsRuntime,
-            CircuitSynchronizationContext synchronizationContext)
+            CircuitSynchronizationContext synchronizationContext,
+            CircuitHandler[] circuitHandlers)
         {
-            Scope = scope ?? throw new ArgumentNullException(nameof(scope));
+            _scope = scope ?? throw new ArgumentNullException(nameof(scope));
```

Comment:
I think at some point we should do cleanup of our DI usage across circuits and Components. We shouldn't be passing DI abstractions around unless the specific class is a composition root.

Reply:
The circuit host is the **owner** of the scope which is why it's here.